fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -16,6 +17,55 @@
 import java.util.List;
 
 public class SqlParser {
+
+    public static List<StatementBase> parse(String originSql, SessionVariable sessionVariable) {
+        List<String> splitSql = splitSQL(originSql);
+        List<StatementBase> statements = Lists.newArrayList();
+        for (int idx = 0; idx < splitSql.size(); ++idx) {
+            String sql = splitSql.get(idx);
+            StatementBase statement = parseSingleSql(sql, sessionVariable);
+            statement.setOrigStmt(new OriginStatement(sql, idx));
+            statements.add(statement);
+        }
+        return statements;
+    }
+
+    public static StatementBase parseSingleSql(String sql, SessionVariable sessionVariable) {
```

Comment:
what the different from other parse(line.69)?

Reply:
The old method only reveive a sqlMode parameter to define the property of  parser. I think there may be more paramter  in session variable for defining parser in the future. So I extract the define logic in a `setParserProperty()` method for expansibility.